File: propms/property_management_solution/report/invoice_details/invoice_details.js
// Copyright (c) 2016, Aakvatech and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Invoice Details"] = {
	"filters": [
		{
			"fieldname":"rental",
			"label": __("Rental"),
			"fieldtype": "Select",
			"options": "Commercial Rent\nResidential Rent",
			"default": "Commercial Rent"
		},
		{
			"fieldname":"maintenance",
			"label": __("Maintenance"),
			"fieldtype": "Check"
		},
		{
			"fieldname":"year",
			"label": __("Year"),
			"fieldtype": "Link",
			"options": "Fiscal Year"
		}
	]
};

File: propms/property_management_solution/report/invoice_details/invoice_details.py
# Copyright (c) 2013, Aakvatech and contributors
# For license information, please see license.txt

from __future__ import unicode_literals

# import frappe
from .other_methods import get_residential_columns
from .other_methods import get_sales_invoice


def execute(filters=None):
    columns, data = [], []
    if filters.get("rental") and filters.get("year"):
        columns = get_residential_columns(filters.get("year"))
        get_sales_invoice(filters, data)

    return columns, data
